### ISSUE TITLE
rl.get_key_pressed() is only firing for some keys

### ISSUE THREAD

**Issue by louisgjohnson** [deltaphc/raylib-rs#41]
The only keys I could get to work with this function:

```
Some(KEY_ONE)
Some(KEY_TWO)
Some(KEY_THREE)
Some(KEY_FOUR)
Some(KEY_FIVE)
Some(KEY_SIX)
Some(KEY_SEVEN)
Some(KEY_EIGHT)
Some(KEY_NINE)
Some(KEY_ZERO)
Some(KEY_MINUS)
Some(KEY_EQUAL)
Some(KEY_EQUAL)
Some(KEY_GRAVE)
Some(KEY_SPACE)
Some(KEY_BACKSLASH)
Some(KEY_RIGHT_BRACKET)
Some(KEY_LEFT_BRACKET)
Some(KEY_SEMICOLON)
Some(KEY_APOSTROPHE)
Some(KEY_SLASH)
Some(KEY_PERIOD)
Some(KEY_COMMA)
```

I'm not sure if this is a core issue or a rust level issue

**Comment by louisgjohnson** [comment]
but things like `rl.is_key_pressed(KEY_W)` works fine

**Comment by yummydev** [comment]
This is a raylib issue. Apparently it will be fixed in 2.6. Until then I recommend rl.get_key_pressed_number

**Comment by Dacode45** [comment]
This is a raylib issue. Apparently it will be fixed in 2.6. Until then I recommend rl.get_key_pressed_number

**Comment by louisgjohnson** [comment]
no worries, thanks for the update @Dacode45
